### ISSUE TITLE
Feature: Support playwright component tests

### ISSUE THREAD

**Issue by cassus** [vitalets/playwright-bdd#57]
**The problem**
I'd like to use bddgen with component tests

The documentation says I should
`import { test as base } from 'playwright-bdd'` but that is

```ts
import { TestInfo, test as base } from '@playwright/test';
export const test = base.extend<BddFixtures>({
  //...
})
```

To use component testing and have the `mount` fixture available I must start with 
`import {test as baseTest} from "@playwright/experimental-ct-react"`

**A solution**

It might be a solution if `src/run/bddFixtures.ts` would export not only `test` as it's defined currently,
but also the object that is currently passed to `base.extend`, so I could call `.extend<BddFixtures>` 
on a different base, including the one I imported from `@playwright/experimental-ct-react` or 
even on bases that I have already extended (so I can share fixtures between by bddgen tests and plain playwright tests)

What do you think @vitalets ?

**Comment by vitalets** [comment]
I like it. Will add components testing support.

**Comment by cassus** [comment]
Seems simple enough for a first contribution. Would you like to get a PR?

**Comment by vitalets** [comment]
> Seems simple enough for a first contribution. Would you like to get a PR?

Sure!!

**Comment by vitalets** [comment]
Released in v5.6.0.
  Please re-check on your side and report if there are any issues.
